### PR TITLE
Backend refactor - Tasks/10302

### PR DESCRIPTION
**Pull request by FuriousLlama**
# Fixes or Implements [AB#10302](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/10302)

* [x] Enhancement
* [ ] Bug
* [ ] Documentation

## Description

A sentences or two describing the overall goals of the pull request's commits.

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

* [x] Unit Tests Updated
* [ ] Functional Tests Updated
* [ ] Not Required

### CODE REVIEW

**Review by tiagograf-AOT — APPROVED** [review]

File: Apps/Common/src/Services/EmailQueueService.cs [hunk 1]
```diff
@@ -35,46 +33,28 @@ namespace HealthGateway.Common.Services
...
-        /// <param name="emailInviteDelegate">Invite email delegate to be used.</param>
         /// <param name="environment">The injected environment configuration.</param>
         public EmailQueueService(
             ILogger<EmailQueueService> logger,
-            IConfiguration configuration,
             IBackgroundJobClient jobClient,
             IEmailDelegate emailDelegate,
-            IMessagingVerificationDelegate emailInviteDelegate,
             IWebHostEnvironment environment)
         {
```
Comment (MikeLyttle) [inline Apps/Common/src/Services/EmailQueueService.cs:53]: 11 errors are being reported in CommonTests/Services/EmailQueueService_Test due to this change:

```
Error	CS1729	'EmailQueueService' does not contain a constructor that takes 6 arguments
```

File: Apps/Common/src/Services/IEmailQueueService.cs [hunk 2]
```diff
@@ -71,37 +71,22 @@ public interface IEmailQueueService
...
-        /// </summary>
-        /// <param name="invite">The populated invite with associated email to send.</param>
-        void QueueNewInviteEmail(MessagingVerification invite);
-
-        /// <summary>
-        /// Queues an email to be sent.
-        /// </summary>
-        /// <param name="inviteEmailId">The id of the email to send.</param>
-        void QueueInviteEmail(Guid inviteEmailId);
-
```
Comment (MikeLyttle) [inline Apps/Common/src/Services/IEmailQueueService.cs:97]: 3 errors are being reported in CommonTests/Services/EmailQueueService_Test due to this change:

```
Error	CS1061	'IEmailQueueService' does not contain a definition for 'QueueNewInviteEmail' and no accessible extension method 'QueueNewInviteEmail' accepting a first argument of type 'IEmailQueueService' could be found
Error	CS1061	'IEmailQueueService' does not contain a definition for 'QueueInviteEmail' and no accessible extension method 'QueueInviteEmail' accepting a first argument of type 'IEmailQueueService' could be found
```

**Review by sslaws — APPROVED** [review]

**Review by MikeLyttle — CHANGES_REQUESTED** [review]

File: Apps/Common/src/Services/EmailQueueService.cs (hunk 1 above)
Comment (FuriousLlama) [reply Apps/Common/src/Services/EmailQueueService.cs:53]: Good catch. I've updated them

File: Apps/Common/src/Services/IEmailQueueService.cs (hunk 2 above)
Comment (FuriousLlama) [reply Apps/Common/src/Services/IEmailQueueService.cs:97]: fixed!

**Review by MikeLyttle — APPROVED** [review]

**Pull request merged**